iconic/UserProfile/views.py

```diff
@@ -19,6 +19,47 @@ def get(self, request):
         return Response(data, status=status.HTTP_200_OK)
 
 
+# A view for reading the number of followers
+class NumberOfFollowersView(APIView):
+    def get(self, request):
+        number_of_followers = Followers.objects.filter(user_id=request.user.id).count()
+        return Response({"user_id": request.user.id, "number_of_followers": number_of_followers},
+                        status=status.HTTP_200_OK)
```

Comment:
You wrote request that returns count of followers of authenticated user, but the task was in request that get user_id and returns count of followers of this user_id user